scalabel/label/coco_typing.py

```diff
@@ -79,3 +79,40 @@ class PredType(TypedDict):
     bbox: List[float]
     image_id: int
     category_id: int
+
+
+class PnpSegType(TypedDict, total=False):
```

Comment:
Pnp seems like a little short as abbreviation for Panoptic Segmentation, let's choose a more clear abbrevation? (Similar to our discussion on idcs vs indices yesterday)

Reply:
Is `pan` OK? I see in the [panopticapi](https://github.com/cocodataset/panopticapi), `pan` is used: https://github.com/cocodataset/panopticapi/blob/118ffaba478b5d45925e88d504625b4123f3160a/panopticapi/evaluation.py#L86

Reply:
We can simply use `Panoptic` here. You can use `pan` for local variables if necessary. The coco API is also called `panopticapi` instead of `panapi` for clarity.

Reply:
Also, Pnp usually means Perspective-n-Point in computer vision.